### PR TITLE
Planning: reduce running time in path_bounds decider.

### PR DESCRIPTION
**Pull request by panjiacheng**

### CODE REVIEW

File: modules/planning/tasks/deciders/path_assessment_decider.cc [hunk 1]
```diff
@@ -41,6 +41,8 @@ using apollo::hdmap::HDMapUtil;
 // PointDecision contains (s, PathPointType, distance to closest obstacle).
 using PathPointDecision = std::tuple<double, PathData::PathPointType, double>;
 
+constexpr double kMinObstacleArea = 1e-4;
```
Comment (xiaoxq) [inline modules/planning/tasks/deciders/path_assessment_decider.cc]: This is a global variable. Mark either `static` or anonymous namespace to make it local to this file.

File: modules/planning/tasks/deciders/path_bounds_decider.cc [hunk 2]
```diff
@@ -245,9 +248,10 @@ std::string PathBoundsDecider::GenerateFallbackPathBound(
   // PathBoundsDebugString(*path_bound);
 
   // 2. Decide a rough boundary based on lane info and ADC's position
-  if (!GetBoundaryFromLanesAndADC(reference_line_info.reference_line(),
+  std::string dummy;
```
Comment (Capri2014) [inline modules/planning/tasks/deciders/path_bounds_decider.cc]: Do not name it dummy, make the name more self-explaining, this string is for "borrow_lane_type" right?

File: modules/planning/tasks/deciders/path_bounds_decider.cc [hunk 3]
```diff
@@ -315,6 +322,38 @@ bool PathBoundsDecider::GetBoundaryFromLanesAndADC(
 
     // 2. Get the neighbor lane widths at the current point.
     double curr_neighbor_lane_width = 0.0;
+    hdmap::Id neighbor_lane_id;
+    if (lane_borrow_info == LaneBorrowInfo::LEFT_BORROW) {
+      // Borrowing left neighbor lane.
+      if (reference_line_info.GetNeighborLaneInfo(
+              ReferenceLineInfo::LaneType::LeftForward,
```
Comment (xiaoxq) [inline modules/planning/tasks/deciders/path_bounds_decider.cc]: `using ...::LaneType;` at the beginning to shorten your code, as it's used many times.

File: modules/planning/tasks/deciders/path_assessment_decider.cc (hunk 1 above)
Comment (panjiacheng) [reply modules/planning/tasks/deciders/path_assessment_decider.cc]: This is a global variable but in .cc only.
Others won't include a .cc file.

Comment (panjiacheng) [reply modules/planning/tasks/deciders/path_assessment_decider.cc]: I added anonymous namespace anyway.

File: modules/planning/tasks/deciders/path_bounds_decider.cc (hunk 3 above)
Comment (panjiacheng) [reply modules/planning/tasks/deciders/path_bounds_decider.cc]: Modified.

File: modules/planning/tasks/deciders/path_bounds_decider.cc (hunk 2 above)
Comment (panjiacheng) [reply modules/planning/tasks/deciders/path_bounds_decider.cc]: Thx. I updated it to be dummy_borrow_lane_type. It is for "borrow lane type" but it is a dummy variable, too.

**Pull request merged**